ideas/99-confidence.md

```diff
@@ -0,0 +1,121 @@
+This section is based on user feedback spawned as part of 75-status-everyday (https://docs.google.com/document/d/1pkfZWxr9I0AqidEuOfogzxEXrcg6ofs2bUkh-HSKD6o/edit)
+
+1. Messages should always be sent.
+
+    a. Generally: lack of trust in messages being sent
+    b. Provide better feedback of when messages are or aren’t sent
+    c. Allow re-send of individual messages a la Whatsapp
+
+2. Messages are in wrong order.
+
+    a. Confirm fixed in 1on1
+    b. Make decision re migrations for old messages
+    c. Understand and clarify work needed for private group / public (placeholders)
+    d. Design: Deal with right order but “message arriving too late” (see sidebar)
+
+3. Messages lack timestamps.
+
+     a. This appears to contribute to a lack of confidence in messaging
+     b. Design (then dev): Add this to UI
+
+4. Messages don’t arrive despite being sent, or they arrive late.
+    a. Disambiguate offline from mail server node copy
+    b. Understand what is needed to make mail server HA
+    c. Give user feedback while it is fetching messages (non-instant)
+    d. Understand if there are more reasons messages would arrive late
+
+5. Messages should always be delivered, 100% of the time.
+
+    a. Similar to 1 but stronger
+    b. What happens if mail server goes down?
+    c. Can we implement read/re-send semantics at Whisper level (mailserver)?
+    d. Look into how read receipts can be improved
+    e. Clarify UI a la Whatsapp double-tick
+    f. Consider tracking send/delivered ratio as a metric
+
+6. Offline inboxing not working in public chat.
+
+    a. Clarify perf dependency on smooth-ui swarm
+    b. While perf is being worked on, educate user about offline only being 1-1
+
+7. Perception of lack of notifications.
+    a. UXR: understand expectation -  for non-contacts? Public chats? in-app?
+    b. Separate PN swarm, clarify dependency with this swarm
+
+8. Get notification but nothing visible in app.
+    a. PN delivery being decoupled from Whisper delivery, see point 7 above
+    b. Clarify dependency on PN v2 swarm and where work done
+
+9. Perception of never received messages from other people.
+    a. Investigate if same as filter while offline bug and current limitations
+    b. More 7: Investigate non-contact PN possibility
+    c. More 7: Public chat PN?
+    d. UXR: Understand this problem and perception better
+
+10. Sometimes need to change mail server manually to get messages working.
+    a. Make failure of individual mail server apparent and prompt switching
+    b. Automated switching?
+    c. How ensure no switching is necessary? (HA, see 4)
+
+### Requirements & Dependencies
+
+Request messages history after background: https://github.com/status-im/status-react/pull/3493
+New protocol https://github.com/status-im/status-react/pull/3273
+https://github.com/status-im/ideas/issues/87
+
+### Minimum Viable Product
```

Comment:
I've seen iteration scope in a few places, google docs, slack PM and slack channel - can we make it more precise what we think is reasonable for next ~1w (before next Town Hall)?